File: Programmers/두 정수 사이의 합.cpp
#include <iostream>
using namespace std;

long long solution(int a, int b) {
	long long answer = 0;
	if (a == b) return a;
	if (a > b) swap(a, b);
	for (a; a <= b; a++)
		answer += a;
	return answer;
}

int main() {
	int a, b;
	cin >> a >> b;
	cout << solution(a, b);
}

File: Programmers/두 정수 사이의 합.java
class Solution {
    public long solution(int a, int b) {
        long answer = Math.min(a, b);
        b = Math.max(a, b);
        a = (int)answer;

        
        for(long i = answer + 1; i <= b; i++)
            answer += i;
        return answer;
    }
}